auto_landing/src/pose.cpp

```diff
@@ -0,0 +1,71 @@
+#include <auto_landing/pose.hpp>
+
+namespace ariitk::auto_landing {
+
+PoseEstimation::PoseEstimation()
+    : height_(3) {}
+
+void PoseEstimation::init(ros::NodeHandle& nh , ros::NodeHandle& nh_private) {
+
+    set_firefly_pose_pub_ = nh.advertise<geometry_msgs::PoseStamped>("command_pose",1);
+    firefly_pose_sub_ = nh.subscribe("quad_odometry",1,&PoseEstimation::quadPoseCallBack,this);
+    firefly_pixel_coordinates_sub_ = nh.subscribe("pixel_coordinates",1,&PoseEstimation::pixelCoordinatesCallBack,this);
+
+    nh_private.getParam("C",camera_to_quad_matrix_);
+    nh_private.getParam("K",camera_matrix_);
+    nh_private.getParam("D",distortion_matrix_);
+    nh_private.getParam("t",tcamera_);
+
+    count_=0;
+    scaleUpMatrix = Eigen::Matrix3f::Zero();
+    setpt_firefly_.pose.position.z = height_;
+    arrayToMatrixConversion();
+}
+
+void PoseEstimation::run() {
+    if(count_%10==0) setptUpdate();
```

Comment:
Remove this if condition. Constantly keeps updating as well as publishing the estimated pose.